tests/tests_ntp_provider7.yml

```diff
@@ -0,0 +1,31 @@
+
+- name: Configure NTP as peer with chrony as specified provider
+  hosts: all
+  vars:
+    timesync_ntp_servers:
+      - hostname: 172.16.123.1
+    timesync_peer_server:
+      - hostname: 127.0.0.1
+    timesync_ntp_provider: chrony
+  roles:
+    - linux-system-roles.timesync
+
+  pre_tasks:
```

Comment:
what's the point of those three tasks?

Reply:
I see:  https://github.com/linux-system-roles/timesync/blob/924650d0cd4117f73a7f0413ab745a8632bc5cec/tests/tests_ntp_provider6.yml#L9